src/ui/src/js/controllers/system_view.js

```diff
@@ -33,8 +41,14 @@ export default function systemViewController(
       ($scope.data.display_name || $scope.data.name),
       $scope.data.version
     );
+    $scope.dtOptions.withLanguage({"info": "Showing _START_ to _END_ of _TOTAL_ entries (filtered from " +
+        $scope.data.commands.length + " total entries)", "infoFiltered":   ""});
   };
 
+  $scope.reloadData = function() {
+     tb = $scope.dtInstance;
```

Comment:
Is this assignment necessary?

Reply:
It is not I was using for testing and thought I removed it.